MPChartLib/proguard-project.txt

```diff
@@ -1,20 +1,4 @@
-# To enable ProGuard in your project, edit project.properties
-# to define the proguard.config property as described in that file.
-#
-# Add project specific ProGuard rules here.
-# By default, the flags in this file are appended to flags specified
-# in ${sdk.dir}/tools/proguard/proguard-android.txt
-# You can edit the include path and order by changing the ProGuard
-# include property in project.properties.
-#
-# For more details, see
-#   http://developer.android.com/guide/developing/tools/proguard.html
+# Whitelist MPAndroidChart
+# Preserve all public classes and methods
 
-# Add any project specific keep options here:
-
-# If your project uses WebView with JS, uncomment the following
-# and specify the fully qualified class name to the JavaScript interface
-# class:
-#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
-#   public *;
-#}
+-keep class com.github.mikephil.charting.** { *; }
```

Comment:
following rules should be used:
`-dontwarn com.github.mikephil.**`

`-keep public class com.github.mikephil.charting.animation.* {
    public protected *;
}`